python/cudf/dataframe/series.py

```diff
@@ -487,6 +487,54 @@ def __rtruediv__(self, other):
 
     __div__ = __truediv__
 
+    def __and__(self, other):
+        """Performs vectorized bitwise and (&) on corresponding elements of two
+        series.
+
+        Difference from pandas:
+        Returned series has type np.result_type(lhs, rhs) instead of bool
```

Comment:
I'm confused by this comment. I checked how Pandas handles these operations and they return the expected type, not bool:

```
>>> a = pd.Series([1,2,3])

>>> b = pd.Series([3,2,1])

>>> a | b
0    3
1    2
2    3
dtype: int64
```

Reply:
Huh! You’re right. I was misled by @mrocklin‘s [SO question](https://stackoverflow.com/questions/22231438/bitwise-operations-in-pandas-that-return-numbers-rather-than-bools)